template/default_tmpl.blade.php

```diff
@@ -1,8 +1,12 @@
 server {
     listen @{{ .Env.LISTEN }} default_server;
-    server_name _;
+@if ($version === 'proxy')
+    server_name  ~^(?<subdomain>.+)\.@{{ .Env.DOMAIN }}$;
```

Comment:
so I like the whole idea, I am just not a big fan of the `subdomain` parsing default behaviour... not sure I can think of something better 🤔